src/renderer/components/ft-share-button/ft-share-button.vue

```diff
@@ -1,14 +1,15 @@
 <template>
   <ft-icon-button
     ref="iconButton"
-    :title="$t('Share.Share Video')"
-    theme="secondary"
+    :title="$t(`Share.Share ${mediaType}`)"
+    :theme="isVideo?'secondary':'base-no-default'"
     :icon="['fas', 'share-alt']"
     dropdown-position-x="left"
     :force-dropdown="true"
   >
     <ft-flex-box>
       <ft-toggle-switch
+        v-if="!isChannel"
```

Comment:
Same for other places
```suggestion
        v-if="isVideo"
```

Reply:
Done !